releases.md

```diff
@@ -0,0 +1,195 @@
+# Express Release Process
+
+This document contains the technical aspects of the Express release process. The
+intended audience is those who have been authorized by the Express Technical
+Committee (TC) to create, promote and sign official release builds for Express,
+as npm packages hosted on https://npmjs.com/package/express.
+
+## Who can make releases?
+
+Release authorization is given by the Express TC. Once authorized, an individual
+must have the following access permissions:
+
+### 1. Github release access
+
+The individual making the release will need to be a member of the
+expressjs/express team with Write or Admin permission level so they are
+able to tag the release commit and push changes to the expressjs/express
+repository (see Steps 4 and 5).
+
+[tunniclm: Is there  a separate way to indicate whether an individual is allowed
+           to tag a release, or some higher level of permission?]
+
+### 2. npmjs.com release access
+
+The individual making the release will need to be a member of the expressjs
+organisation on npmjs.com so they are able to publish the release package
+(see Step 6).
```

Comment:
AFAIK, npm orgs only work for packages scoped under that org's name, but all our modules are top-level modules, which don't work under orgs, right? If we can use an npm org, I'd be all for it. Someone should ask npm to see.